modules/skybrowser/src/targetbrowserpair.cpp

```diff
@@ -151,13 +151,21 @@ double TargetBrowserPair::verticalFov() const {
     return _browser->verticalFov();
 }
 
-std::vector<int> TargetBrowserPair::selectedImages() const {
+std::vector<std::string> TargetBrowserPair::selectedImages() const {
     return _browser->selectedImages();
 }
 
 ghoul::Dictionary TargetBrowserPair::dataAsDictionary() const {
     glm::dvec2 spherical = targetDirectionEquatorial();
     glm::dvec3 cartesian = skybrowser::sphericalToCartesian(spherical);
+    SkyBrowserModule* module = global::moduleEngine->module<SkyBrowserModule>();
+    std::vector<std::string> selectedImagesIndices;
+    
+    for (const std::string& imageUrl : selectedImages()) {
+        selectedImagesIndices.push_back(
+            module->wwtDataHandler().image(imageUrl).value().identifier
```

Comment:
Can do:
`module->wwtDataHandler().image(imageUrl)->identifier`

maybe also a:
`ghoul_assert(module->wwtDataHandler().image(imageUrl).has_value(), "...");`
above juuust to be sure that the image actually exists?